test/integration/voldemort/performance/benchmark/Benchmark.java

```diff
@@ -642,6 +651,18 @@ public static void main(String args[]) throws IOException {
               .withRequiredArg()
               .describedAs(LOCAL_SERVER_PROPERTIES)
               .ofType(String.class);
+        parser.accepts(WORKLOAD_CLASS, "workload class (full name)")
+              .withRequiredArg()
+              .describedAs(WORKLOAD_CLASS)
+              .ofType(String.class);
```

Comment:
yeah.... that sometimes will confuse. Using reflection without providing a way to load the class path will confuse. 

I suggest giving WORKLOAD_TYPE default is default if given File, the other workload is used.